service/src/main/java/tds/session/repositories/impl/SessionRepositoryImpl.java

```diff
@@ -40,7 +59,42 @@ public SessionRepositoryImpl(DataSource datasource) {
             sessionOptional = Optional.empty();
         }
 
-
         return sessionOptional;
     }
+
+    /**
+     * In order to preserve compatibility with the existing TDS system, this implementation executes an UPDATE against
+     * the existing record in the {@code session.session} table.  Once the new code has complete responsibility for
+     * managing a {@link Session}, this implementation will be updated to comply with the "immutability" database design
+     * concept.
+     *
+     * @param sessionId The id of the {@link Session} to pause
+     * @param newStatus The status of the {@link Session} is being paused
+     */
+    @Override
+    public void pause(final UUID sessionId, final String newStatus) {
```

Comment:
Is there ever a case where the pause() method takes in a status that is not "paused"?  Seems like maybe it should be hard-coded in pause and if there is a case that passes in "closed" then we have this as a private helper that excepts it as a string and 2 main public methods pause() and close() that accept a sesisonId only.